kubeletclient/kubeletclient_test.go

```diff
@@ -115,6 +115,15 @@ var _ = Describe("Kubelet resource endpoint data read operations", func() {
 			_, err := GetResourceClient()
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		It("should fail with missing file", func() {
+			// os.Mkdir("tempdir", 0644)
+			kubeletSocket = "Andrew Bays"
```

Comment:
Good example to us but not so universal :laughing:  We may use simple string for that, like "sampleSocketString".